python/lsst/validate/base/measurement.py

```diff
@@ -302,19 +328,23 @@ def check_spec(self, name):
         return self.metric.check_spec(self.quantity, name,
                                       filter_name=self.filter_name)
 
+    def __eq__(self, other):
+        return (self.value == other.value) and (self.metric == other.metric)
 
-class DeserializedMeasurement(MeasurementBase):
+
+class DeserializedMeasurement(Measurement):
     """Measurement deserialized from JSON.
 
+    TODO: do we still need this one?
```

Comment:
Good question. Probably not. My goal was for it to be possible to rebuild a Measurement from its serialized JSON output. The `DeserializedMeasurement` was a compromise because I realized I couldn't rebuild the exact same Blob class (with compute methods and such) from its JSON alone.

tl;dr It's safe to delete this and we'll figure out how deserialize `validate_drp` classes from their JSON serializations.